### DISCUSSION TITLE
Using Prebuilds Images in Other Repos

### DISCUSSION THREAD

**Discussion by metaskills** [community/community#50008] [Codespaces]
### Select Topic Area

Question

### Body

I'd like to use a prebuilt Codespace as an image in another codespace to compose services. Was it ever the case that prebuilt images were available in ghcr.io? Is that still the case? I could not see where they are stored or by what naming convention.

**Comment by bjbennet** [comment]
Hi @metaskills, we do not maintain a library of prebuild repositories. Prebuilds are intended to be consumed in the repository in which they are created. Users primarily will create a prebuild to speed up the creation process of larger, more complex repositories. More on Prebuilds [here](https://docs.github.com/en/codespaces/prebuilding-your-codespaces/about-github-codespaces-prebuilds).

**Reply by metaskills** [reply]
Yes, I know about prebuilds. Been using them for a long time. What I would like to do is use that prebuilt image in another repo via a devcontainer compose. But I can not see where it is stored.

**Reply by cwndrws** [reply]
Hey @metaskills we don't currently support the ability to use prebuilds in this way. It sounds like you'd be better off using an action (or some other mechanism) to build a docker image that is then used by the devcontainer of both repos as the base image. Does that make sense?

**Reply by metaskills** [reply]
It does, thank you. Exploring using this project now. https://github.com/devcontainers/ci I've used it for CI/CD test and deploys, but gonna give it a whirl on making a prebuilt image too for dev.
